### PR TITLE
Export CeramicCliUtils

### PR DESCRIPTION
**Pull request by ukstv**
This pull request mainly exports CeramicCliUtils. Additionally, and that was necessary, it adds missing dependencies. For some packages, a dependency was used, but not declared in `package.json`.

### CODE REVIEW

File: packages/ceramic-doctype-account-link/src/account-link-doctype.ts [hunk 1]
```diff
@@ -10,6 +10,10 @@ export interface AccountLinkParams extends DocParams {
     content?: object;
 }
 
+function isValidChainId(chainId: string) {
+    return chainId === 'eip155:1' || chainId === 'fil:t' || chainId === 'fil:m'
+}
```
Comment (oed) [inline packages/ceramic-doctype-account-link/src/account-link-doctype.ts:15]: We should really support all caip-2 chainIDs that are supported by `3id-blockchain-utils`. So I don't think we need to check if it's valid here. We can just accept all of them!

**Review by oed — CHANGES_REQUESTED** [review]
Looks good. Just one thing I think we should remove.

File: packages/ceramic-doctype-account-link/src/account-link-doctype.ts (hunk 1 above)
Comment (ukstv) [reply packages/ceramic-doctype-account-link/src/account-link-doctype.ts]: Yay! Makes total sense.

**Review by oed — APPROVED** [review]
Looks good to me! Please have a look @simonovic86

File: packages/ceramic-core/src/anchor/anchor-service-factory.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ export class AnchorServiceFactory {
      * Get AnchorService instance by type
      */
     public get(): AnchorService {
+        console.log('anc', this._config.anchorServiceUrl)
```
Comment (simonovic86) [inline packages/ceramic-core/src/anchor/anchor-service-factory.ts:23]: do we need this `console.log`?

**Review by simonovic86 — CHANGES_REQUESTED** [review]

File: packages/ceramic-core/src/anchor/anchor-service-factory.ts (hunk 2 above)
Comment (ukstv) [reply packages/ceramic-core/src/anchor/anchor-service-factory.ts:23]: Sure as hell no. Sorry for that. And thank you for finding this.

Comment (simonovic86) [reply packages/ceramic-core/src/anchor/anchor-service-factory.ts]: awesome! no worries! 😊

**Review by simonovic86 — APPROVED** [review]

**Pull request merged**